src/component/bom.js

```diff
@@ -1296,6 +1296,8 @@ const DependenciesTabHeader = React.memo(({
   isComponentLoading,
   componentRefs,
   getSpecialComponentFeature,
+  specialComponentId,
+  isAddedByUser,
```

Comment:
nit: `was` (added by user)? or just omit prefix (addedByUser)? we mgiht also name this `browserlocal-only´ or similar (this more closely reflects the contract, as such changes are not user- but browser-specific - users might have multiple browsers..)

that being said, it would of course be nice to also offer persistent user-profiles.... but that's for another pullrequest, I suppose